lib/backend/etcdv3/etcdv3.go

```diff
@@ -85,6 +87,17 @@ func NewEtcdV3Client(config *apiconfig.EtcdConfig) (api.Client, error) {
 func (c *etcdV3Client) Create(ctx context.Context, d *model.KVPair) (*model.KVPair, error) {
 	logCxt := log.WithFields(log.Fields{"model-etcdKey": d.Key, "value": d.Value, "ttl": d.TTL, "rev": d.Revision})
 	logCxt.Debug("Processing Create request")
+
+	// Add in the UID and creation timestamp for the resource if needed since etcd does not do it itself.
```

Comment:
Not everything that gets here is a resource - so you can't cast to an ObjectMetaAccessor.

Any reason why you're not doing this processing in resources.go?